### ISSUE TITLE
Convert to celsius?

### ISSUE THREAD

**Issue by johwe091** [brycesub/silvia-pi#13]
How do I change the unit to celcius? Do I only need to delete the line where you change it to farenheit and then change allowed input?

Thanks for a great project!

**Comment by obiben** [comment]
You`d also need to change the settemp in config, and the cold/warm definitions

**Comment by brycesub** [comment]
Indeed, if you write a patch to make C or F configurable I'd merge it.
